scripts/daemon_rfid_reader.py

```diff
@@ -9,11 +9,23 @@
 
 print dir_path
 
+
+base_path = os.path.dirname(dir_path)
+
+try:
```

Comment:
@marcohorstmann 
isn't this always overwriting these files with an empty string each time a card is swiped?